src/metadata.json.in

```diff
@@ -1,5 +1,5 @@
 {
-    "shell-version": ["3.22", "3.24"],
+    "shell-version": ["3.20", "3.22", "3.24"],
```

Comment:
The problem here is that I don't have the time to test it on versions other than the one I'm running. 

It would be great if you could do the testing for 3.20 in the future!